code/modules/mob/living/basic/space_fauna/spider/young_spider/young_spider.dm

```diff
@@ -0,0 +1,174 @@
+/**
+ * # Young Spider
+ *
+ * A mob which can be created by spiderlings/spider eggs.
+ * The basic type is the guard, which is slow but sturdy and outputs good damage.
+ * All spiders can produce webbing.
+ */
+/mob/living/basic/young_spider
```

Comment:
One day, this and giant spiders (maybe spiderlings too) should probably share a `/mob/living/spider` parent type, though this is not that day.